generated/nidaqmx/_grpc_interpreter.py

```diff
@@ -3133,11 +3135,12 @@ def set_trig_attribute_string(self, task, attribute, value):
                 task=task, attribute_raw=attribute, value=value))
 
     def set_trig_attribute_timestamp(self, task, attribute, value):
+        ts = GrpcTimestamp()
         response = self._invoke(
             self._client.SetTrigAttributeTimestamp,
             grpc_types.SetTrigAttributeTimestampRequest(
                 task=task, attribute_raw=attribute,
-                value=convert_time_to_timestamp(value)))
+                value=convert_time_to_timestamp(value,ts)))
 
```

Comment:
`convert_time_to_timestamp` returns None, so `value=convert_time_to_timestamp(...)` does not do anything useful.

Feel free to change the signature of `convert_time_to_timestamp` to `def convert_time_to_timestamp(dt: Union[std_datetime, ht_datetime]) -> GrpcTimestamp` if that's what you need. Then the calling template doesn't need to declare a local variable `ts`.